docs/content/en/docs/concepts/_index.md

```diff
@@ -23,13 +23,28 @@ read the configuration file from the current directory.
 | `apiVersion` | The Skaffold API version you would like to use. The current API version is {{< skaffold-version >}}. |
 | `kind`  |  The Skaffold configuration file has the kind `Config`.  |
 | `build`  |  Specifies how Skaffold should build artifacts. You have control over what tool Skaffold can use, how Skaffold tags artifacts and how Skaffold pushes artifacts. Skaffold supports using local Docker daemon, Google Cloud Build, Kaniko, or Bazel to build artifacts. See [Using Builders](/docs/how-tos/builders) and [Using Taggers](/docs/how-tos/taggers) for more information. |
-| `test` |  Specifies how Skaffold should test artifacts. Skaffold supports [container-structure-tests](https://github.com/GoogleContainerTools/container-structure-test) to test built artifacts.See [Using testers](/docs/how-tos/testers) for more information. |
-| `deploy` |  Specifies how Skaffold should deploy artifacts. Skaffold supports using `kubectl`, Helm, or kustomize to deploy artifacts.See [Using Deployers](/docs/how-tos/deployers) for more information. |
+| `test` |  Specifies how Skaffold should test artifacts. Skaffold supports [container-structure-tests](https://github.com/GoogleContainerTools/container-structure-test) to test built artifacts. See [Using testers](/docs/how-tos/testers) for more information. |
+| `deploy` |  Specifies how Skaffold should deploy artifacts. Skaffold supports using `kubectl`, Helm, or kustomize to deploy artifacts. See [Using Deployers](/docs/how-tos/deployers) for more information. |
 | `profiles`|  Profile is a set of settings that, when activated, overrides the current configuration. You can use Profile to override the `build` and the`deploy`> section. |
 
 You can learn more about the syntax of `skaffold.yaml` at
 [`skaffold.yaml References`](https://github.com/GoogleContainerTools/skaffold/blob/master/examples/annotated-skaffold.yaml).
 
+## Global configuration (~/.skaffold/config)
+
+Some context specific settings can be configured in the global configuration file `~/.skaffold/config`. You can configure the options for specific or any context.
```

Comment:
This file location is just the default, but it can theoretically be anywhere. Let's change this to something like

```Some context specific settings can be configured in a global configuration file, defaulting to `~/.skaffold/config`.```

Reply:
Also, last line:

```Options can be configured globally or for specific contexts.```

Reply:
Sounds much better :+1: